src/script.h

```diff
@@ -13,4 +15,7 @@ int script_get_push_opcode_size_from_bytes(
     size_t bytes_len,
     size_t *size);
 
+/* Get OP_N */
+bool is_op_n(unsigned char op, bool allow_zero, size_t *n);
```

Comment:
should be prefixed with `script_` to avoid name clashes.

Reply:
Done